packages/suite/src/views/dashboard/components/PortfolioCard/index.tsx

```diff
@@ -40,15 +40,18 @@ const Wrapper = styled.div`
 `;
 
 const PortfolioCard = memo(() => {
-    const { fiat, localCurrency } = useFiatValue();
+    const { coins, localCurrency } = useFiatValue();
     const { discovery, getDiscoveryStatus, isDiscoveryRunning } = useDiscovery();
     const accounts = useFastAccounts();
     const { dashboardGraphHidden } = useSelector(s => s.suite.flags);
     const dispatch = useDispatch();
 
     const isDeviceEmpty = useMemo(() => accounts.every(a => a.empty), [accounts]);
+    const isMissingFiatRates = coins
+        .filter(c => !c.tokenAddress)
+        .some(c => !c.current?.rates?.[localCurrency]);
```

Comment:
this does not work for testnet coins as they have rates === undefined so if you have any testnet active, you see loading forever

Reply:
301cf708065475f8f7baef48bd54d27929e63336